tests/test_numba_wrappers.py

```diff
@@ -0,0 +1,125 @@
+"""
+Tests the numba wrapper functions against the equivalent pyssht functions
+using random input data.
```

Comment:
I'm okay with using random input as long as there are no unintended side-effects related to the stochasticity. It might be good to have some tests that use known input for some things (if that's not done already).

Reply:
I think this is a good idea, and we should make an issue for it (I didn't touch the tests in this PR other than to do formatting etc.)